text/0000-self-documenting-buildpacks.md

```diff
@@ -0,0 +1,46 @@
+# Meta
+[meta]: #meta
+- Name: Self Documenting Buildpacks
+- Start Date: 1-23-2020
+- CNB Pull Request: (leave blank)
+- CNB Issue: (leave blank)
+- Supersedes: (put "N/A" unless this replaces an existing RFC, then link to that RFC)
+
+# Summary
+[summary]: #summary
+
+This RFC suggest a mechanism where buildpacks can provide documentation for the benefit of their consumers.
+
+# Motivation
+[motivation]: #motivation
+
+At the moment the origin and provinence of a buildpack are are sort of a black box. Users have asked where to find information about the repos for buildpacks so they can file issues, but we would also like a method to in-situ allow buildpacks to surface their configuration options or anything else their authors think pertinent
+
+# What it is
+[what-it-is]: #what-it-is
+
+Would add an additional, optional, field to `buildpack.toml` -  `docs`.
```

Comment:
In order for information to be available to `pack inspect-builder`, it will need to be added to builder/bpkg labels. This RFC should include those labels, especially if they necessitate a change to the distribution spec. We could use a `buildpack.toml` fields to populate the label(s).